torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,513 @@
+
+    .. note::
+       This dataset requires the following additional library to be installed:
+
+       * `pandas <https://pypi.org/project/pandas/>`_ to load CSV files
+
+    .. versionadded:: 0.3
+    """
+
+    BAND_SETS: dict[str, tuple[str, ...]] = {
+        "all": (
+            "VV",
+            "VH",
+            "B2",
+            "B3",
+            "B4",
+            "B5",
+            "B6",
+            "B7",
+            "B8",
+            "B8A",
+            "B11",
+            "B12",
+            "2020_Map",
+            "avg_rad",
+        ),
+        "s1": ("VV", "VH"),
+        "s2": ("B2", "B3", "B4", "B5", "B6", "B7", "B8", "B8A", "B11", "B12"),
+    }
+
+    modality_urls = {
+        "esa_wc": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/esa_wc/ESA_WC.zip"  # noqa: E501
+        },
+        "viirs": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/viirs/VIIRS.zip"  # noqa: E501
+        },
+        "mask": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/mask/mask.zip"  # noqa: E501
+        },
+        "s1": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s1/s1_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s1/s1_part2.zip",  # noqa: E501
+        },
+        "s2_temporal_subset": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_temporal_subset/s2_temporal_subset_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_temporal_subset/s2_temporal_subset_part2.zip",  # noqa: E501
+        },
+        "s2_autumn": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_autumn/s2_autumn_part2.zip",  # noqa: E501
+        },
+        "s2_spring": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_spring/s2_spring_part2.zip",  # noqa: E501
+        },
+        "s2_summer": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_summer/s2_summer_part2.zip",  # noqa: E501
+        },
+        "s2_winter": {
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part1.zip",  # noqa: E501
+            "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/s2_winter/s2_winter_part2.zip",  # noqa: E501
+        },
+    }
+
+    split_url = "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/split_IDs/split_IDs.csv"  # noqa: E501
```

Comment:
Might make more sense to have a single url:
```python
url = "https://huggingface.co/datasets/burakekim/mapinwild/resolve/main/"
```
and then a set/list of relative paths to the actual files. Then you won't have so many long lines and it will be easier to change if you ever move your dataset somewhere else.

Reply:
Ah, nice catch!